Gems/ScriptCanvas/Code/Include/ScriptCanvas/AutoGen/ScriptCanvas_Nodeable_Macros.jinja

```diff
@@ -11,10 +11,10 @@ SPDX-License-Identifier: Apache-2.0 OR MIT
 {# Macros for Nodeables #}
 
 {# CleanName #}
-{# Returns a clean name without quotations, commas, slashes or white space #}
+{# Returns a clean name without quotations, commas, or white space #}
 
 {%- macro CleanName(name) -%}
-{{name|replace(' ','')|replace('"','')|replace('/','')|replace(':','')}}
+{{name|replace(' ','')|replace('"','')|replace(':','')}}
```

Comment:
There don't seem to be any other nodeables with slashes in their name. It makes sense to remove double quotes, but I am not sure why this macro needs to remove slashes, colons, commas, or even spaces for node names in case people want to include those characters in the name. I will likely remove it altogether when I get to cleaning up the jinja templates. At worst, this would only affect the name of the nodeable that appears in the editor, not the name that is used internally. So it wouldn't break existing nodeables